### DISCUSSION TITLE
Messages are too far apart from each other

### DISCUSSION THREAD

**Discussion by romeogiggletoes** [Chatterino/chatterino2#3460] [Ideas]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

Below you can see that there is a quite noticeable amount of space between messages. Even when emote scaling is set below the threshold where it creates extra spacing, I find that there is too much spacing which triggers my auts.

### Screenshots

![chatterino message spacing](https://user-images.githubusercontent.com/97059214/148290335-bdb1dc85-51b0-46ef-a4e2-d4120d4da416.png)

_No response_

### OS and Chatterino Version

Chatterino Nightly + Dankerino patches 2.3.4 (commit 3044f4a5d, ) on Windows 10

**Comment by ALazyMeme** [comment]
This is a suggestion, not an issue. Please carefully read and follow the issue templates - which would've lead you to creating a discussion:
![image](https://user-images.githubusercontent.com/12804673/148039950-c92471a9-93f5-41b9-b4a0-170e9527bab4.png)

**Reply by romeogiggletoes** [reply]
I've also updated this one, I figured it's essentially impossible to fix the spacing when scaling emotes too large. Let me know what you think 🤔

**Comment by romeogiggletoes** [comment]
I think an option for messages to be able to specify how much spacing between them, with the ability to also overlap them is ideal. This can be done by having a drop down menu where you can also type a custom value.  
![chatterino drop down menu](https://user-images.githubusercontent.com/97059214/148289788-00bece95-92da-45f2-8132-16920f95feaf.png)

My goal would be to make chat look closer like this: 
![2022_01_05_15_44_25_Messages_are_too_far_apart_from_each_other_especially_with_high_value_emote_sca](https://user-images.githubusercontent.com/97059214/148289564-2524c6b6-eb02-4042-99a2-36d5f8436210.png)

With smaller emote scaling; normal is on left, edited is right: (retook this image with properly sized badges)
![2022_01_05_18_52_53_Messages_are_too_far_apart_from_each_other_Discussion_3460_Chatterino_chatt](https://user-images.githubusercontent.com/97059214/148321325-50c509dd-3a2b-4e9a-8042-628391d94a46.png)

**Reply by romeogiggletoes** [reply]
~~Also when editing I found that the badges are a little bit bigger than the text, so maybe resizable badges or just straight up making them a bit smaller could be a useful addition.~~ Figured I can just increase font sizes and decrease interface zoom to work around the badge sizes, spacing between messages though is still an issue.
